src/plugins/data/common/index_patterns/constants.ts

```diff
@@ -15,3 +15,16 @@ export const RUNTIME_FIELD_TYPES = [
   'boolean',
   'geo_point',
 ] as const;
+
+/**
+ * Used to determine if the instance has any user created index patterns by filtering index patterns
+ * that are created and backed only by Fleet server data
+ * Should be revised after https://github.com/elastic/kibana/issues/82851 is fixed
+ * For more background see: https://github.com/elastic/kibana/issues/107020
+ */
+export const FLEET_ASSETS_TO_IGNORE = {
+  LOGS_INDEX_PATTERN: 'logs-*',
+  METRICS_INDEX_PATTERN: 'metrics-*',
+  LOGS_DATA_STREAM_TO_IGNORE: 'logs-elastic_agent', // ignore ds created by Fleet server itself
+  METRICS_DATA_STREAM_TO_IGNORE: 'metrics-elastic_agent', // ignore ds created by Fleet server itself
```

Comment:
Now that we're not checking doc counts, we need to add the `metrics-endpoint.metadata_current_default` index to this list since there may be indices created but no data flowing yet. Note this is not a data stream, but an index created by a transform.

Reply:
Thanks for the details and testing steps 👍